client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -431,6 +435,7 @@ export const JetpackSitesDataViews = ( {
 		actions: [],
 		setDataViewsState: setDataViewsState,
 		dataViewsState: dataViewsState,
+		onSelectionChange: ( [ item ]: SiteData[] ) => openSitePreviewPane( item.site.value ),
```

Comment:
What value is this adding on the a4a side?

Reply:
Adding this can keep the selection item  consistent between the Dataviews component and Calypso